### ISSUE TITLE
raw output export

### ISSUE THREAD

**Issue by Canop** [Canop/bacon#240]
With the export framework, it would be trivial to implement an export writing the raw command output in a file, to be done either on end of task or on a key press.

I'll wait for a demand to implement it, though.

**Comment by Bolognafingers** [comment]
I'd love it as it'd enable me to use bacon inside Cursor/VSCode

**Comment by Canop** [comment]
Why ? Some people already use bacon inside vscode, in the integrated terminal.

**Comment by screwyprof** [comment]
Hey @Canop. I've got a question. How can I copy all the output? So I've got a target to run tests with `nextest` analyser, and I can toggle raw output to see the results (because otherwise the analyser will show that the test failed, but what show why it happened), and now I want to copy the output to show it to an AI assistant and I can't do it. Because  ctrl+c only copies the current visible part of the frame, and not the contents of the whole frame.

**Comment by Canop** [comment]
@screwyprof You'd like to be able to copy the content of the raw output to the clipboard, probably with a key shortcut ?

**Comment by screwyprof** [comment]
@Canop yes, exactly. Let me elaborate on it. In the past I used to use `cargo watch`. It's definitely has less features, but it was very easy to copy the output. 

The reason I need it is to show it to the AI agents so that they could change the code based on the messages. If it helps at the moment I'm working on a [pet project](https://github.com/screwyprof/favkit) and recently added `bacon` to it. 

So now I have bacon open inside VSCode. It builds the projects, runs linters and tests. So when something fails I want to be able to copy the output to show it to `Cursor AI` so that it could fix it. 

I guess soon enough the AI will be smart enough to be able to run a command and fix the errors based on the output on its own.

**Comment by Canop** [comment]
@screwyprof Made a dedicated issue for your need: https://github.com/Canop/bacon/issues/282
